app/src/main/java/com/example/habitsmasher/ui/history/EditHabitEventDialog.java

```diff
@@ -92,11 +112,35 @@ public void onClick(View view) {
                         eventComment,
                         _editHabitEvent.getId());
                 _errorText.setText("");
-                _habitEventListFragment.updateListAfterEdit(editedHabitEvent,_index);
+                _habitEventListFragment.editHabitEvent(editedHabitEvent,_index, _selectedImage);
 
                 // Close dialog
                 getDialog().dismiss();
             }
         });
     }
+
+    private void fetchEventImageFromDB() {
+        FirebaseStorage storage = FirebaseStorage.getInstance();
+        StorageReference storageReference = storage.getReference();
+
+        StorageReference ref = storageReference.child("images/" + _userId + "/" + _parentHabit.getId() + "/" + _editHabitEvent.getId() + "/eventImage");
+
+        final long ONE_MEGABYTE = 1024 * 1024;
```

Comment:
This is named as a constant, but appears as a local variable? Let's pick one option instead.

- [ ] constant or local variable?

Reply:
update?